Table: add defaultRowAction (#298)
BSI-Business-Systems-Integration-AG/org.eclipse.scout.docs: docs/modules/releasenotes/partials/release-notes-25.2.adoc

The new `AbstractRemoveDoEntityContributionValueMigrationHandler` can be used when a data object contribution class is completely deleted. +
This migration handler will remove the contribution with the given type name and type version from the data objects.

[[table-default-row-action]]
== Table default row action

> The Scout JS `Table` has now the property `defaultRowAction` of the type `Action`.

Reviewer: ```suggestion
The Scout JS `Table` has a new property `defaultRowAction` of type `Action`.
```

> This `Action` is executed when a row is double-clicked.

Reviewer: ```suggestion
If set, this action is executed when a row is double-clicked.
```

> Prior to the execution a `rowAction` event is triggered.

Reviewer: ```suggestion
Prior to the execution, a `rowAction` event is triggered.
```

> When default is prevented on this event the `defaultRowAction` is not executed.

Reviewer: ```suggestion
When default is prevented on this event, the `defaultRowAction` is not executed.
```

> The `defaultRowAction` can be used execute a particular `Menu` of the `Table` on a row action.

Reviewer: ```suggestion
The `defaultRowAction` can be used to execute a particular `Menu` of the `Table` on a row action.
```
